gitkraken-client/commit-signing-with-gpg.md

```diff
@@ -161,25 +164,42 @@ Open a Terminal and run this command:
 
 <img src="/wp-content/uploads/gkc-ssh-keygen.png" srcset="/wp-content/uploads/gkc-ssh-keygen@2x.png 2x" class="img-bordered img-responsive center">
 
-####Configure git to sign commits with SSH
 
-Run this command to use SSH for signing commits and pointing to the key previously created:
-`git config --global gpg.format sshgit config --global user.signingkey /PATH/TO/.SSH/KEY.PUB`
+####Enable Git Executable feature
+
+Go to <kbd>Preferences > Experimental > Git Executable</kbd> and enable it.
+
+<img src="/wp-content/uploads/gkc-git-executable.png" srcset="/wp-content/uploads/gkc-git-executable@2x.png 2x" class="img-bordered img-responsive center">
+
+####Select SSH as your GPG format for signing
+
+[Which program to select](https://git-scm.com/docs/git-config#Documentation/git-config.txt-gpgltformatgtprogram)
+
+At <kbd>Preferences > GPG > GPG Format</kbd>, select <kbd>SSH</kbd>.
+
+Automatically Gitkraken client will change your preferences in `.gitconfig` and populate GPG SSH Program with ssh-keygen
+
+####Select the signing key
+
+[How to select SSH signing key](https://git-scm.com/docs/git-config#Documentation/git-config.txt-usersigningKey)
```

Comment:
For all these links (How to select SSH signing key, Allowed Signers file help, Which program to select), can we please make these into complete sentences i.e. See this [documentation] for more information on [how to select ssh signing key || working with the allowed signers file || which program to select].